aggregate/housing_security/rent_stable_three_maintenance.py

```diff
@@ -7,23 +8,22 @@
 from utils.PUMA_helpers import borough_name_mapper, clean_PUMAs
 
 suffix_mapper = {
-    "_N": "",
+    "_N": "_count",
```

Comment:
I am all about adding the `_count` token here, but it needs to be added for variance of count as well. so the first three columns for three or more maintenance should be 

```
'units_threemaintenance_count', 'units_threemaintenance_count_cv', 'units_threemaintenance_count_moe'
```